src/Servers/Kestrel/test/InMemory.FunctionalTests/ResponseTests.cs

```diff
@@ -2285,6 +2286,58 @@ public async Task NoResponseSentWhenConnectionIsClosedByServerBeforeClientFinish
             }
         }
 
+        [Fact]
+        public async Task AppAbortIsLogged()
+        {
+            var testContext = new TestServiceContext(LoggerFactory);
+
+            using (var server = new TestServer(httpContext =>
+            {
+                httpContext.Abort();
+                return Task.CompletedTask;
+            }, testContext))
+            {
+                using (var connection = server.CreateConnection())
+                {
+                    await connection.Send(
+                        "GET / HTTP/1.1",
+                        "Host:",
+                        "",
+                        "");
+                    await connection.ReceiveEnd();
+                }
+                await server.StopAsync();
+            }
+
+            Assert.Single(TestApplicationErrorLogger.Messages.Where(m => m.Message.Contains("The connection was aborted by the application.")));
```

Comment:
Nit: Can we get these error messages from the resx/core strings directly so we don't need to update two places?